.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+  "trailingComma": "es5",
```

Comment:
Shouldn't it be ES6 as [Datopian uses ES6 as style reference](https://playbook.datopian.com/style-guide/javascript/#_1-formatting)?

Reply:
@cuducos don't worry about this 👍 It's just related to only trailing comma. According to [Prettierr docs it puts trailing comma where it is valid in ES5](https://prettier.io/docs/en/options.html#trailing-commas)